subsys/periphconf/Kconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025 Nordic Semiconductor
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Periphconf integration
+
+config NRF_PERIPHCONF_MIGRATE
+	bool "Generate periphconf_migrated.c"
+	default y
+	depends on SOC_NRF54H20_CPURAD
+	depends on SOC_NRF54H20_CPUAPP
+	# NB: Note that PPR and FLPR are in the global domain, and
+	# generation of periphconf_migrated.c is only supported for local
+	# domains.
```

Comment:
Would it be better to place this in the help text? IMO this is valuable information for the user, which typically use some menuconfig tool and is not necessarily looking at the kconfig source.